client/src/components/generic/Header.js

```diff
@@ -70,7 +70,7 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-export const Header = () => {
+export const Header = ({hideEmployers=false}) => {
```

Comment:
Nitpick: I think that our styling standards suggest spaces on the edges of ({ this="kind of statement" })